npm-shrinkwrap.json

```diff
@@ -10166,7 +10166,7 @@
       "from": "git://github.com/vladikoff/node-uap.git#9cdd16247",
       "requires": {
         "array.prototype.find": "2.0.0",
-        "uap-core": "git://github.com/ua-parser/uap-core.git#108591a790f2a52685d694846bd43e8ce1f70adb",
```

Comment:
Can you revert the changes to npm-shrinkwrap.json? Since you didn't update any of the dependencies, this file shouldn't need to be updated.

Reply:
Yes, I'll do that!